tests/scripts/run_demos.py

```diff
@@ -1,44 +1,57 @@
 #!/usr/bin/env python3
 """Run the Mbed TLS demo scripts.
 """
+import argparse
 import glob
 import subprocess
 import sys
 
-def run_demo(demo):
+def run_demo(demo, quiet=False):
     """Run the specified demo script. Return True if it succeeds."""
-    returncode = subprocess.call([demo])
+    args = {}
+    if quiet:
+        args['stdout'] = subprocess.DEVNULL
+        args['stderr'] = subprocess.DEVNULL
```

Comment:
I'm not sure about suppressing stderr. On one hand, it could be useful in case somethings fails, on the other hand it's probably not going to be too helpful without context from stdout, and most likely in case of errors we'll want to run the demo that failed anyway.

Reply:
If you suppress stdout without stderr, some working demos will show some errors out of context (e.g. a signature failure without the preceding “and now here's what happen if you corrupt the file before asking for a signature check”).